src/upstream.c

```diff
@@ -402,9 +417,7 @@ void *processUpstreamMessage()
 								{
 									ParodusError("Failed to get device_id\n");
 								}
-						}
-						else
-						{
+						} else if (WRP_MSG_TYPE__SVC_ALIVE != msgType) {
```

Comment:
Add comment that keep alive message should not be sent to cloud.

Reply:
OK